ext/cache/sqlite.rs

```diff
@@ -1,12 +1,12 @@
 // Copyright 2018-2025 the Deno authors. MIT license.
+use std::future::poll_fn;
```

Comment:
```suggestion

use std::future::poll_fn;
```

(blank line to prevent automatic usings from going above the copyright header)